RomanASP.py

```diff
@@ -71,6 +72,9 @@ def main():
                         help='Number of images to use with SN detections')
     # TODO:change all instances of this variable to det_images
 
+    parser.add_argument('-o', '--output_path', type=str, required=False,
```

Comment:
For a later PR, but I think `output_path` is output directory.  To me, a `path` includes the filename.